src/main/java/org/jabref/gui/preferences/external/ExternalTab.fxml

```diff
@@ -21,6 +21,10 @@
         <Label text="%Subject for sending an email with references"/>
         <TextField fx:id="eMailReferenceSubject" minWidth="150.0" HBox.hgrow="ALWAYS"/>
     </HBox>
+    <HBox alignment="CENTER_LEFT" spacing="10.0">
+        <Label text="%Send-to-kindle email address"/>
```

Comment:
Maybe `Email address for sending to Kindle`? In this way, the term what to input comes first ("Email address") and then the concretization ("sending to Kindle")